src/webview/addConnection.ts

```diff
@@ -0,0 +1,45 @@
+// @ts-nocheck
```

Comment:
Please don't use `ts-nocheck`. If you absolutely must, use a [type assertion](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#type-assertions) on an individual value. (For example, you could assert the output of `acquireVsCodeApi` to the type of the `vscode` module).

Reply:
The following should allow type coverage over `vscode`:

```suggestion
import type { Webview } from 'vscode';
declare var acquireVsCodeApi: () => Webview;
```

You can get type coverage over `document` by adding the entry `"DOM"` to the `"lib"` key in `tsconfig.json`.

This will uncover a few previously unchecked type errors!